src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package duke.task;
+
+import duke.parser.DateTimeParser;
+import java.time.LocalDate;
+
+public class Deadline extends Task {
+    protected String type;
+    protected LocalDate time;
+
+    public Deadline(String name, LocalDate time) {
+        super(name);
+        this.time = time;
+        this.type = "D";
+    }
+
+    @Override
+    public String toString() {
+        String status = this.isDone ? "X" : " ";
+        return "[" + this.type + "][" + status + "]" + this.name + " (by: "
+                + DateTimeParser.formatDate(this.time) + ")";
```

Comment:
according to code standard the breaks should occurs outside the parenthesized expression
longName1 = longName2 * (longName3 + longName4 - longName5)
        + 4 * longname6